application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -235,10 +234,10 @@ private RestAction<Message> dmUser(MessageChannelUnion sourceChannel, ForumPost
         String jumpUrlSuffix = " ";
 
         String messageForDm = messageTemplate.formatted("", " on " + guild.getName(),
-                forumPost.message.getJumpUrl() + jumpUrlSuffix);
+                forumPost.forumPost().getMessage().getJumpUrl() + jumpUrlSuffix);
 
         String messageOnDmFailure = messageTemplate.formatted(" " + forumPost.author.getAsMention(),
-                "", forumPost.message.getJumpUrl() + jumpUrlSuffix);
+                "", forumPost.forumPost().getMessage().getJumpUrl() + jumpUrlSuffix);
```

Comment:
Could we move this `forumPost.forumPost().getMessage().getJumpUrl()+ jumpUrlSuffix` to a seperate local variable, something like `postUrl` ? Keep in mind `jumpUrlSuffix` is meant to prevent discord from posting unnecessary preview.

Tbh im not sure if `forumPost` is a good enough name anymore since it contains `forumPost` itself now. But that local var `postUrl` should at least make it more clear at least when it's used.